project/MiMa.scala

```diff
@@ -1025,6 +1025,50 @@ object MiMa extends AutoPlugin {
         // #20553 Tree flattening should be separate from Fusing
         ProblemFilters.exclude[MissingClassProblem]("akka.stream.Fusing$StructuralInfo"),
         ProblemFilters.exclude[MissingClassProblem]("akka.stream.Fusing$StructuralInfo$")
+      ), 
+      "2.4.14" -> Seq(
```

Comment:
I'm actually constantly confused by this Map... I thought this is "from which version the difference was made" so "since 2.4.14 these things" kind of. Do I misunderstand this Map @2m ?